services/movie.py

```diff
@@ -0,0 +1,36 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> Movie:
```

Comment:
The return type for `get_movies` function should be a `QuerySet` instead of `Movie`, because the function is returning a queryset of movies, not a single movie instance. Please, correct the return type.